src/traces/scattercarpet/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var ScatterCarpet = {};
+
+ScatterCarpet.attributes = require('./attributes');
+ScatterCarpet.supplyDefaults = require('./defaults');
+// ScatterCarpet.colorbar = require('../scatter/colorbar');
```

Comment:
I guess should _just work_ as `scattercarpet` is using the same colorscale attribute as the cartesian version.